modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/utils/date_time/format.ts

```diff
@@ -140,7 +140,6 @@ const FORMATTER_MAP = {
 			hour: '2-digit',
 			minute: '2-digit',
 			month: '2-digit',
-			second: '2-digit',
```

Comment:
@bryceosterhaus We checked `'P p'` is only used by us in Commerce, but there seems to be an inconsistency between `p` (hour and minute) and `P` (day, month and year), such that `'P p'` should be the combination of the previous two. Instead we found also `seconds` and @monaco-fabio proposes to remove it, also to accommodate our tests. What do you think?

Reply:
Hey! yeah if that is the case, feel free to change that since it is only used in commerce

Reply:
Thanks a lot Bryce 👍🏼